application/views/emsl_template/globals.php

```diff
@@ -10,10 +10,18 @@
     <meta name="theme-color" content="#ffffff">
 
       <!-- Base jquery -->
-      <script src="/resources/scripts/jquery/jquery-1.11.2.js" type="text/javascript"></script>
+      <!-- <script src="/resources/scripts/jquery/jquery-1.11.2.js" type="text/javascript"></script> -->
```

Comment:
I'd remove the comment so the scanner doesn't get confused

Reply:
Agreed. I also added in a directive at the top to get phpcs to ignore the file since there's no real logic in it other than the if statement.